### PR TITLE
fix(grid): apply debug styling to all columns

### PR DESCRIPTION
**Pull request by austin94**
## Description

* The `styles` property in the package.json was invalid. This PR removes that entry.
* When consumed through `css-modules` the `is-debug` styling was invalid for all columns that had a number or breakpoint value. This adds a wildcard attribute to apply to ALL `col` classes within it. 

## Checklist

* [x] :ok_hand: style updates are Garden Designer approved (add the
  designer as a reviewer)
* [x] :globe_with_meridians: component demo is up-to-date (`yarn start`)
* [x] :white_check_mark: all component states are represented
  (`.is-hovered`, `.is-focused`, etc.)
* [x] :arrow_left: renders as expected with reversed (RTL) direction
* [x] :metal: renders as expected sans Bedrock (`?bedrock=false`)
* [x] :nail_care: provides `custom.css` example for modifying the
  primary accent color
* [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: packages/grid/package.json [hunk 1]
```diff
@@ -35,6 +37,5 @@
   ],
   "publishConfig": {
     "access": "public"
-  },
-  "style": "src/index.css"
```
Comment (jzempel) [inline packages/grid/package.json]: @austin94 why did you remove this? It informs `postcss-import` where to load from if someone decides to use `@import '@zendeskgarden/css-grid'`.

Comment (austin94) [reply packages/grid/package.json]: @ryanseddon did some research and mentioned that postcss-import will default to `main` if there is no `style` provided. Also, that file doesn't exist in `src/`

Comment (jzempel) [reply packages/grid/package.json]: Oh right. Unlike the other packages, this is Sass source code. So no `postcss-import`.
